### PR TITLE
Feature/sheet view add notes

### PR DESCRIPTION
**Pull request by deonjo**
Screenshot 2019-07-11 15 05 01

Added a column for notes for people in sheet view.

### CODE REVIEW

**Comment by deonjo** [comment]
This let's you add notes for people. I'm going to extend it, so that they can put information like address, and year/age, gender. It's all the information that you need to arrange people. Notes is just something general.

File: src/containers/listView/sheetView.jsx [hunk 1]
```diff
@@ -318,8 +318,6 @@ class SheetView extends Component {
                             rowHeaders={false} 
                             afterChange={this.onCellValueChange}
                             minSpareRows={1}
-                            contextMenu={this.state.contextMenu}
```
Comment (jeffchiucp) [inline src/containers/listView/sheetView.jsx]: why are we removing the dropdown menu?

**Comment by jeffchiucp** [comment]
wondering why we remove the ag-Grid & drop-down menu?

**Comment by jeffchiucp** [comment]
Screen Shot 2019-07-11 at 11 24 49 PM
Screen Shot 2019-07-11 at 11 22 32 PM

**Comment by jeffchiucp** [comment]
@deonjo wondering why we remove the drop-down menu?

**Comment by deonjo** [comment]
We removed AG-Grid, because AG-Grid's community version is limited. You can't do copy and paste. And we have switched completely over to Handsontable instead now.

I removed the dropdown context menu, because i'm not yet ready to support the actions in the dropdown menu.

**Pull request merged**